treeherder/webapp/api/job_log_url.py

```diff
@@ -45,23 +38,3 @@ def list(self, request, project, jm):
         job_log_url_list = jm.get_job_log_url_list([job_id])
 
         return Response(job_log_url_list)
-
-    @detail_route(methods=['post'])
-    @with_jobs
-    def parse(self, request, project, jm, pk=None):
-        """Trigger an async task to parse this log."""
-
-        log_obj = jm.get_job_log_url_detail(pk)
-        job = jm.get_job(log_obj["job_id"])[0]
-
-        log_obj["job_guid"] = job["job_guid"]
-        # We can fake the result because there is only one queue for high priority
-        # jobs and it doesn't depend on the result
-        log_obj["result"] = None
-
-        logger.info("{0} has requested to parse log {1}".format(
-                    request.user.username, log_obj))
-
-        jm.schedule_log_parsing([log_obj], priority="high")
```

Comment:
This is the only place where the default `priority` is overridden. As such `schedule_log_parsing()` can be simplified (along with then `parse_job_logs()`and the list of queues in settings.py, the Procfile and bin scripts). However I don't mind if you file a separate bug for this.